wdl/model/draft2/src/main/scala/wdl/draft2/model/WdlSyntaxErrorFormatter.scala

```diff
@@ -9,6 +9,10 @@ import scala.collection.JavaConverters._
 
 case class WdlSyntaxErrorFormatter(terminalMap: Map[Terminal, WorkflowSource]) extends SyntaxErrorFormatter {
 
+  // Hashing the terminalMap is very expensive because we re-hash the entire workflow once for every terminal in it (exponential with size of workflow)
+  // To approximate, just use Java's case-class-ignorant implementation (hash functions are allowed to be imperfect)
+  override def hashCode(): Int = System.identityHashCode(this)
```

Comment:
Remember to also override `equals` when overriding `hashCode` and vice versa

Reply:
I'm not sure how to implement this subject to the constraint that [equal objects must have equal hashcodes](https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#equals(java.lang.Object)) - now that the hashcode no longer considers the case class fields

Reply:
I think object equality in scala is just ‘x eq y’

Reply:
Ah, OK - have put that in.

My understanding is that hashcode is supposed to be computed based on a subset of the state used for equals - which is now true here, because both use identity.

However, I feel like `equals` is now wrong / less useful because a deep comparison of two WDL case class hierarchies may now unexpectedly fail. I'm wondering if a `hashCode` consistent with the default `equals` wouldn't serve us better.